src/libLookup/Lookup.cpp

```diff
@@ -1653,17 +1657,18 @@ bool Lookup::ProcessSetDSInfoFromSeed(const bytes& message, unsigned int offset,
     LOG_GENERAL(INFO, "[" << PAD(i++, 3, ' ') << "] " << ds.second);
   }
 
-  if (m_mediator.m_blocklinkchain.GetBuiltDSComm().size() !=
-      m_mediator.m_DSCommittee->size()) {
-    isVerif = false;
+  if (m_mediator.m_blocklinkchain.GetBuiltDSComm().size() != dsNodes.size()) {
     LOG_GENERAL(
-        WARNING,
-        "Size of " << m_mediator.m_blocklinkchain.GetBuiltDSComm().size() << " "
-                   << m_mediator.m_DSCommittee->size() << " does not match");
+        WARNING, "Size of "
+                     << m_mediator.m_blocklinkchain.GetBuiltDSComm().size()
+                     << " " << dsNodes.size() << " does not match");
+    return false;
   }
 
+  bool isVerif = true;
+
   for (i = 0; i < m_mediator.m_blocklinkchain.GetBuiltDSComm().size(); i++) {
-    if (!(m_mediator.m_DSCommittee->at(i).first ==
+    if (!(dsNodes.at(i).first ==
```

Comment:
Change ! == to !=

Reply:
We don't have `!=` implemented for pubkey, so that would require a change to libCrypto as well.  But yes would be good to have that soon in another PR.